reference/enterprise-api-ref/sql-schema.markdown

```diff
@@ -500,14 +508,15 @@ Information about communication between CFEngine clients.
 
 * **RemoteHostKey** *(text)*
     `HostKey` of the remote host.
+
 * **RemoteHostIP** *(text)*
     IP address of the remote host.
 
 * **LastSeenTimeStamp** *(timestamp)*
     Time when the connection was established.
 
 * **LastSeenInterval** *(real)*
-    Frequency within which both hosts open connection.
+    Average frequency in minutes between connections (both INCOMING and OUTGOING) with the host.
```

Comment:
After fix in master this is in expressed in **seconds**, incoming and outgoing end up in separate entries (LastSeenDirection column), interval is for specific direction.